static/css/mozsearch.css

```diff
@@ -430,7 +439,7 @@ tr.after-context-line + tr.before-context-line {
 
 .expando {
   cursor: pointer;
-  color: black;
+  color: var(--page-foreground);
```

Comment:
Can this just be currentColor or inherit instead?

Reply:
Apparently not. Inherit/currentColor gets the color from a .left-column element which also holds line numbers and is colored #aaa.